packages/nextra-theme-docs/src/components/search.tsx

```diff
@@ -117,17 +117,56 @@ export function Search({
   )
 
   const finishSearch = () => {
-    if (input.current) {
-      input.current.value = ''
-      input.current.blur()
-    }
+    input.current?.blur()
     onChange('')
     setShow(false)
     setMenu(false)
   }
 
+  const mounted = useMounted()
   const renderList = show && !!value
 
+  const icon = (
+    <Transition
+      show={mounted && (!show || !!value)}
+      as={React.Fragment}
+      enter="transition-opacity"
+      enterFrom="opacity-0"
+      enterTo="opacity-100"
+      leave="transition-opacity"
+      leaveFrom="opacity-100"
+      leaveTo="opacity-0"
+    >
+      <kbd
+        className={cn(
+          'absolute ltr:right-1.5 rtl:left-1.5 my-1.5 select-none',
+          'rounded bg-white px-1.5 h-5 font-mono font-medium text-gray-500 text-[10px]',
+          'border dark:bg-dark/50 dark:border-gray-100/20',
+          'contrast-more:border-current contrast-more:text-current contrast-more:dark:border-current',
+          'items-center gap-1 transition-opacity',
+          value
+            ? 'cursor-pointer hover:opacity-70 z-20 flex'
+            : 'hidden sm:flex pointer-events-none'
+        )}
+        title={value ? 'Clear' : undefined}
+        onClick={() => {
+          onChange('')
+        }}
+      >
+        {value
+          ? 'ESC'
+          : mounted &&
+            (navigator.userAgent.includes('Macintosh') ? (
+              <>
+                <span className="text-xs">⌘</span>K
+              </>
+            ) : (
+              '/'
```

Comment:
I think we can do CTRL+K for non-Mac browsers

Reply:
<img width="392" alt="image" src="https://user-images.githubusercontent.com/7361780/185007549-74b239e4-d5c3-465c-a896-b830c51135f9.png">